### ISSUE TITLE
Sponsor logo accessibility

### ISSUE THREAD

**Issue by terotik** [kausaltech/kausal-watch-ui#34]
Make sure sponsor logo on footer has accessible description (image has text, text should be available as plain text to all users)

WCAG 2.1  1.1.1  A
